packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -245,6 +245,11 @@ fi
 %endif
 
 %changelog
+* Thu Oct 25 2018 Yunhee Lee <yhlee@amazon.com> - 1.22.0-1
+- Add support for ECS Secrets integrating with AWS Systems Manager Parameter Store
```

Comment:
I think this is only for ecs init change log

Reply:
updated!